smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -148,9 +176,12 @@ function getUserSelectedDevicesCallback(devices) {
 }
 
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize = function() {
-  loadDeviceList();
```

Comment:
(But, probably, it would make much more sense to move this logging into the background script - it's not useful when closing the window stops this information from logging.)